site/collection.tmpl

```diff
@@ -160,7 +160,7 @@
                 {{ end }}
               </td>
               <td class="cell-tags">
-                {{ range .Tags }}<div class="gh-tag tag-{{ .ID }}" title="{{ .Desc }}">{{ .ID }}</div> {{ end }}
+                {{ range $k, $_ := .Tags }}<div class="gh-tag tag-{{ $k.ID }}" title="{{ $k.Desc }}">{{ $k.ID }}</div> {{ end }}
```

Comment:
Ahh, now I remember why I did this: consistent sorting of tags during display. I see that the Go developers did add sorting to template iteration though, so maybe this isn't a problem after all:

https://github.com/golang/go/commit/a5950df89e129e93f76aa8391f84beb460ef82be

If it is, we can always add a slice representation later. Thanks!

Reply:
was a concern but yep, should be able to do one sort to slice right before template generation👍